src/components/error-summary/error-summary.test.js

```diff
@@ -28,4 +28,44 @@ describe('Error Summary', () => {
     const moduleName = await page.evaluate(() => document.activeElement.dataset.module)
     expect(moduleName).toBe('error-summary')
   })
+
+  let inputTypes = [
+    // [description, input id, selector for label or legend]
+    ['an input', 'input', 'label[for="input"]'],
+    ['a textarea', 'textarea', 'label[for="textarea"]'],
+    ['a select', 'select', 'label[for="select"]'],
+    ['a date input', 'dateinput-day', '.test-date-input-legend'],
+    ['a specific field in a date input', 'dateinput2-year', '.test-date-input2-legend'],
+    ['a file upload', 'file', 'label[for="file"]'],
+    ['a group of radio buttons', 'radios', '#test-radios legend'],
+    ['a group of checkboxes', 'checkboxes', '#test-checkboxes legend'],
+    ['a single checkbox', 'single-checkbox', 'label[for="single-checkbox"]'],
+    ['a conditionally revealed input', 'yes-input', '#test-conditional-reveal legend']
+  ]
+
+  describe.each(inputTypes)('when linking to %s', async (_, inputId, legendOrLabelSelector) => {
+    beforeAll(async () => {
+      await page.goto(`${baseUrl}/examples/error-summary`, { waitUntil: 'load' })
```

Comment:
Not super-keen on hooking into the review app's examples, but couldn't think of a better way of doing this at the minute. This is something I'd definitely to revisit in the future.

Reply:
Yeah this is an issue with all our examples like this.